config/environments/development.rb

```diff
@@ -34,7 +34,21 @@
   # Adds additional error checking when serving assets at runtime.
   # Checks for improperly declared sprockets dependencies.
   # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
+
+config.assets.raise_runtime_errors = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.delivery_method = :smtp
+  
+  config.action_mailer.smtp_settings = {
+  address:              'smtp.gmail.com',
+  port:                 '587',
+  domain:               'gmail.com',
+  :user_name          => 'demo.botree@gmail.com',
+  :password           => 'India@123!',
```

Comment:
Never add confidential details directly like this. Use it through environment variables. for example `ENV['DEMO_EMAIL']`